IntegrationTests/tests_04_performance/test_01_resources/test_udp_1000_reqs_1_conn.swift

```diff
@@ -0,0 +1,23 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2020 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import NIO
+
+func run(identifier: String) {
+    measure(identifier: identifier) {
+        let numberDone = try! UdpShared.doUdpRequests(group: group, number: 1000)
```

Comment:
I think you could pull the `Channel` setup out of the `measure` block and this would become much more stable. The TCP tests possibly get that wrong. When the alloc tests were first written this wouldn't have been possible :)

Reply:
As discussed.  If possible I'd like to keep everything in.  Will give some grace in the checked numbers to make it stable in error reporting at least.

Reply:
Decided to leave it in there for now but give a little grace in numbers.

Reply:
Grace seems to make things stable.